### ISSUE TITLE
Clarify library usage

### ISSUE THREAD

**Issue by nerdstein** [GetDKAN/dkan#2164]
1. Define a list of all of the third party libraries we're using. We should acknowledge that we're using Drupal and would inherit it's licenses.

2. For each of the libraries we're using, we need to identify each license and capture the library and license in the LICENSE.lib file. 

3. OF NOTE: LICENSE.txt file can leave the GPLv2 license for this code in the repo. 

4. Update the README file to reference both the LICENSE.txt file for dkan code and reference LICENSE.lib for dependent libraries

cc: @marctjones

**Comment by marctjones** [comment]
License.LIb file should take the format of something similar to
""""
---
DKAN use the <x.1 library> under the terms of the <y.1 license> as described below.

<text of y.2 license>

---
DKAN use the <x.2 library> under the <y.2 license> as described below:

<text of y.2 license>
---

Would be cool if we could autobuild this based on list of packages.

**Comment by dafeder** [comment]
I'm not sure what the scope of this should really be. I think this is fairly obsolete as an issue; composer makes this all a lot more accessible and there are tools like [dominikb/composer-license-checker](https://github.com/dominikb/composer-license-checker) if we want to add some automation (I ran that against DKAN 2.x and found no conflicting licenses). Closing this, we can open a new issue if this question re-emerges with clearer requirements.
